homeassistant/components/html5/manifest.json

```diff
@@ -2,9 +2,11 @@
   "domain": "html5",
   "name": "HTML5 Push Notifications",
   "codeowners": [],
```

Comment:
Thank you for your contribution thus far! 🎖 Since this is a significant contribution, we would appreciate you'd added yourself to the list of code owners for this integration. ❤️ 

Please, add your GitHub username to the `manifest.json` of this integration.

For more information about "code owners", see: [Architecture Decision Record 0008: Code owners](https://github.com/home-assistant/architecture/blob/master/adr/0008-code-owners.md).

Reply:
Done